src/app/view/components/login/forgot-password-component/forgot-password.component.ts

```diff
@@ -53,28 +73,20 @@ export class ForgotPasswordComponent implements OnInit {
      */
     private loadLabels(): void {
         this.loginService.getLoginFormInfo(this.language, this.i18nMessages).subscribe(
```

Comment:
maybe we can do this in just one place and not repeat this in several components